src/librustc_yk_sections/mir_cfg.rs

```diff
@@ -255,7 +256,9 @@ impl<'a, 'tcx> PhiInserter<'a, 'tcx> {
             a_orig.resize(blocks.len(), Vec::default());
             for (a, def_blks) in self.def_sites.iter().enumerate() {
                 for bb in def_blks {
-                    a_orig[*bb].push(u32::try_from(a).unwrap())
+                    // The `def_sites` vector is guaranteed to have no more entries than a `u32`
```

Comment:
We can shorten this to something like e.g. "def_sites is guaranteed to have at most u32::max_value() items".